src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+           --data '{"name" : "<folder_name>", "parent_id": "root"}'
+    ~~~
+
+    Replace:
+    - `<folder_name>`: a name for the folder.
+    - `<api-key>`: your service account's API key.
+
+    The response includes the new folder's ID.
+
+1. To create a descendent folder within a folder, set `parent_id` to the folder's ID:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/folders' \
+           --header 'Authorization: Bearer <api-key>' \
+           --data '{"name" : "<folder_name>", "parent_id": "<parent_id>"}'
+    ~~~
+
+    Replace:
+    - `<folder_name>`: a name for the folder.
+    - `<api-key>`: your service account's API key.
+    - `<parent-id>`: the ID of the folder that will contain the new folder.
+
+1. To grant your service account the `FOLDER_ADMIN` role on a folder and its descendent folders:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<service-account-id>/FOLDER/<folder-id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
+    - `<service-account-id>`: The ID of the service account you created in [Gather Required Information](#gather-required-information).
+    - `<folder-id>` with the ID of the folder.
+    - `<api-key>`: your service account's API key.
+
+    {{site.data.alerts.callout_info}}
+    You can also grant the role to a human user. However, during Limited Access, only a service account using the {{ site.data.products.db }} API or Terraform provider can manage folders.
+    {{site.data.alerts.end}}
+
+1. If you grant the role at the level of the organization, it is inherited on all folders. To give your service account the `FOLDER_MOVER` role at the organization level:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<service-account-id>/ORGANIZATION/<organization-id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
+    - `<service-account-id>`: your service account's ID.
+    - `<organization-id>` with the ID you gathered in [Gather Required Information](#gather-required-information).
+    - `<api-key>`: your service account's API key.
+
+To access a folder's clusters, a user or service account must also have the Cluster Administrator, Cluster Creator, or Cluster Operator role on the folder. The role may be granted by inheritance or directly on a cluster.
+
+## List folder contents
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- To list descendent folders: [Org Administrator](authorization.html#org-administrator), [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover).
```

Comment:
So this part is pretty important: cluster roles grant you permission to view folders. So I would rework this section. 

To list the children of a folder, you must have a folder role OR cluster role on the target folder. E.g. if you have Cluster Developer on Folder A, you can view Folder A AND all descendants.

Reply:
Done